tutorials/hist/twoscales.py

```diff
@@ -43,7 +43,7 @@
 hint1.Scale(scale)
 hint1.Draw("same")
 
-axis = ROOT.TGaxis(ROOT.gPad.GetUxmax(),ROOT.gPad.GetUymin(),
+axis = ROOT.TGaxis(ROOT.gPad.GetUxmin(),ROOT.gPad.GetUymin(),
```

Comment:
why? your change will draw a diagonal line, instead of a secondary vertical Y axis

Reply:
Are you maybe thinking of https://root-forum.cern.ch/t/automatic-repositioning-of-tgaxis-after-zooming/61906 ?